app/Command/Compile.hs

```diff
@@ -154,6 +154,8 @@ targetParser =
       $ maybe (Opts.readerError targetsMessage) pure
       . flip M.lookup targets
       . T.unpack
+      . T.takeWhile (/= ' ')
```

Comment:
How about using http://hackage.haskell.org/package/text-1.2.3.0/docs/Data-Text.html#v:strip?

Reply:
Thanks, couldn't find that when I looked through the docs before!